Source/WebCore/css/parser/CSSParserToken.cpp

```diff
@@ -661,19 +665,28 @@ void CSSParserToken::serialize(StringBuilder& builder, const CSSParserToken* nex
         }
         break;
     case NumberToken:
-        // These won't properly preserve the NumericValueType flag.
-        if (m_numericSign == PlusSign)
-            builder.append('+');
-        builder.append(numericValue());
+        if (mode == SerializationMode::CustomProperty)
```

Comment:
Same, it looks like it could be a `if constexpr`

Reply:
But it is not a template parameter?

Reply:
It seems it could be. The only time it's called with `SerializationMode::CustomProperty` is statically determined in `CSSVariableData::serialise()`.

Reply:
I don't think templating this is a meaningful optimization.

Reply:
Yes indeed serialisation is really not perf critical to the point of avoiding a runtime `if` I guess 👌